index.html

```diff
@@ -9,11 +9,71 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <!-- header starts here-->
+<header id="header">  
+   <nav id="nav-bar"> 
```

Comment:
```suggestion
<header class="header">  
   <nav class="nav-bar"> 
```
Normally, software teams use classes for styling and not IDs. The same applies for the rest of this file